akka-cluster/src/multi-jvm/scala/akka/cluster/LargeMessageClusterSpec.scala

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.cluster
+
+import scala.concurrent.duration._
+
+import akka.actor.ActorIdentity
+import akka.actor.ActorRef
+import akka.actor.Identify
+import akka.actor.PoisonPill
+import akka.cluster.ClusterEvent.UnreachableMember
+import akka.remote.testconductor.RoleName
+import akka.remote.testkit.MultiNodeConfig
+import akka.remote.testkit.MultiNodeSpec
+import akka.testkit._
+import com.typesafe.config.ConfigFactory
+
+object LargeMessageClusterMultiJvmSpec extends MultiNodeConfig {
+  val first = role("first")
+  val second = role("second")
+  val third = role("third")
+
+  // Note that this test uses default configuration,
+  // not MultiNodeClusterSpec.clusterConfig
+  commonConfig(ConfigFactory.parseString(
+    """
+    akka {
+      loglevel = INFO
+      cluster.debug.verbose-heartbeat-logging = on
```

Comment:
you can enable this to get some more logging of sent and received heartbeats, on DEBUG log level